src/main/java/org/groomUniv/meet/oauth/enums/BodyType.java

```diff
@@ -0,0 +1,5 @@
+package org.groomUniv.meet.oauth.enums;
+
+public enum BodyType {
+마름,보통,퉁퉁,근육질
+}
```

Comment:
```suggestion
    THIN("마름"),
    NORMAL("보통");

    private final String description;
```
이런식으로 작성하면 가독성이 좋아질 것 같아요

Reply:
네넵 모든 이넘타입 그런식으로 수정하겠습니다